kubevirt-vmware/pkg/controller/v2vvmware/client.go

```diff
@@ -139,6 +140,10 @@ func NewClient(ctx context.Context, credentials *LoginCredentials) (*Client, err
 
 	log.Info(fmt.Sprintf("NewClient, user: '%s', host: '%s'", credentials.username, credentials.host))
 
+	if strings.IndexFunc(credentials.host, unicode.IsSpace) >= 0 {
```

Comment:
What about trimming it first? That should not cause any semantics issues.

Reply:
it depends on what we want to do:
1. fail fast and let user know that s/he has a faulty secret (which might be used by other places) 
2. be lenient and trim it first

thoughts?

Reply:
I think we should trim

Reply:
ok, done